addons/ui/fnc_initDisplayPassword.sqf

```diff
@@ -1,4 +1,6 @@
 #include "script_component.hpp"
+// since 2.19 "server:port" is stored on the display in "guid" variable, makes password filling compatible with "connectToServer" command
+#define GET_SERVER (if (productVersion#2 > 218) then {_display getVariable "guid"} else {_ctrlServerList lbData lbCurSel _ctrlServerList})
```

Comment:
do we need this for the profiling branch (will still be 218)?

Reply:
```suggestion
#define GET_SERVER (_display getVariable ["guid", _ctrlServerList lbData lbCurSel _ctrlServerList])
```